run_experiments.sh

```diff
@@ -0,0 +1,23 @@
+#! /bin/bash
```

Comment:
put this into the `tests/experiment` folder

Reply:
Done